### PR TITLE
Update Hazard 5.5x2 for season 4

### PR DESCRIPTION
**Pull request by jieyouxu**

### CODE REVIEW

File: difficulties/Hazard_5.5x2_60_Nitra.cd.json [hunk 1]
```diff
@@ -94,6 +104,50 @@
...
+      "SpawnAmountModifier": 0.5,
+      "MinSpawnCount": 2
+    },
+    "ED_Terminator": {
+      "Base": "ED_Terminator",
+      "DifficultyRating": 280,
+      "MaxSpawnCount": 1,
+      "SpawnAmountModifier": 0.5
+    }
+  },
```
Comment (trumank) [inline difficulties/Hazard_5.5x2_60_Nitra.cd.json]: What is the reason behind tweaking descriptors? 5.5 is a vanilla difficulty and vanilla difficulties with a multiplier generally only change enemy count, enemy cap, and resupply cost. A difficulty changing anything else should be named something else.

Comment (jieyouxu) [reply difficulties/Hazard_5.5x2_60_Nitra.cd.json]: (I have no clue tbh, whoeven sent me this file and asked me to help submit a change)
I should remove these descriptor changes.

Comment (jieyouxu) [reply difficulties/Hazard_5.5x2_60_Nitra.cd.json]: Removed. Let me go ask whoeven what the intention was.
EDIT: they don't know either, so I'll just remove it.

**Comment by trumank** [comment]
Dynamic `MaxActiveSwarmers` and `MaxActiveEnemies` are pretty inconsistent between similar difficulties (looking at 6x2, 5x2, and 5.5x2), probably because I changed some to be dynamic and forgot to do the rest. `[120, 180, 180, 180]` looks reasonable for 5.5x2 and should probably be applied to 5x2 since they are basically the same.

**Pull request merged**